lib/Group.php

```diff
@@ -217,4 +217,28 @@ protected function _location_in( $param, $value ) {
 		);
 	}
 
+	/**
+	 * Multiple location rule builder for equality checks.
+	 *
+	 * @param  string $param Parameter.
+	 * @param  array  $value Value required for validation.
+	 *
+	 * @return array         Location rule.
+	 *
+	 * @since   1.1.0
+	 */
+	protected function _location_are( $param, $values ) {
+		$location = array();
+		foreach ($values as $value) {
```

Comment:
WP PHP Standards. Spaces around every set of values inside `()`. On this line, on line 233 and 239.